QuoteAPI.go

```diff
@@ -0,0 +1,139 @@
+package arn
+
+import (
+	"errors"
+
+	"github.com/aerogo/aero"
+	"github.com/aerogo/api"
+	"reflect"
+)
+
+// Force interface implementations
+var (
+	_ Likeable      = (*Quote)(nil)
+	_ Publishable   = (*Quote)(nil)
+	_ api.Newable   = (*Quote)(nil)
+	_ api.Editable  = (*Quote)(nil)
+	_ api.Deletable = (*Quote)(nil)
+)
+
+// Actions
+func init() {
+	API.RegisterActions("Quote", []*api.Action{
+		// Publish
+		PublishAction(),
+
+		// Unpublish
+		UnpublishAction(),
+
+		// Like
+		LikeAction(),
+
+		// Unlike
+		UnlikeAction(),
+	})
+}
+
+// Create sets the data for a new quote with data we received from the API request.
+func (quote *Quote) Create(ctx *aero.Context) error {
+	user := GetUserFromContext(ctx)
+
+	if user == nil {
+		return errors.New("Not logged in")
+	}
+
+	quote.ID = GenerateID("Quote")
+	quote.Created = DateTimeUTC()
+	quote.CreatedBy = user.ID
+	quote.Likes = []string{}
+
+	return quote.Unpublish()
+}
+
+// Edit remove the quote from it's previous linked character quote list if the new one is different
+// and add it to the new one.
+func (quote *Quote) Edit(ctx *aero.Context, key string, value reflect.Value, newValue reflect.Value) (bool, error) {
+	if key == "CharacterId" {
+		newCharacterId := newValue.String()
+		previousCharacterId := newValue.String()
+		if previousCharacterId != newCharacterId {
```

Comment:
There should be an empty line here before the start of an `if` block.
Also don't forget the `ID` spelling on the variables.